### PR TITLE
Create page

### PR DESCRIPTION
**Pull request by philipchurchley**

### CODE REVIEW

**Review by jennweir — APPROVED** [review]
Great start to the create page. I created an issue #13 so we do not forget to make the buttons connect to the backend. Default function stubs can be left for now.
Also note: naming conventions should be more specific instead of "my-dropdown". Keep this in mind and make changes accordingly. Still approved PR since this can be fixed with the next iteration of changes.

File: application/templates/create.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            </div>
+        </div>
+    </div>
+
+
+    <div class="row">
+        <div class="column">
+            <h2>+ Add Schedule</h2>
+            <div class="calendar">
+                <!--you will need to import a different link because this links to my google calendar lol-->
```
Comment (reikoc1735) [inline application/templates/create.html:57]: to add on to this i think we should have ppl be able to input their google calendar (but this is just a place holder for now)

**Review by reikoc1735 — APPROVED** [review]

File: application/templates/create.html (hunk 1 above)
Comment (jennweir) [reply application/templates/create.html:57]: Awesome, thanks for the feedback. I will add it to #13

**Review by tinglu12 — APPROVED** [review]
Looks good

**Pull request merged**